src/methods/geom_relations/geom_geom_processors.jl

```diff
@@ -211,6 +211,7 @@ end
 the line (ls to le) and β is the traction of the curve (cs to ce). =#
 function _find_intersect_fracs(ls, le, cs, ce)
     _, fracs = _intersection_point(
+        Float64,
         (_tuple_point(ls), _tuple_point(le)),
```

Comment:
What do you mean by everything else? Does this mean to not ignore the first output of `_intersection_point` and then use that in lines 221, 223, and 225?

Reply:
I think @rafaqz means that we should save the `_tuple_point` outputs into a variable and then pass those tuples into all of the below functions rather than passing `ls`, `le`, `cs`, and `ce`. We could call the variables `t_ls`, `t_le`, ...

Reply:
I'm look at all the functions called in the while loop in line 128. Out of these, it seems only `segment_segment_orientation` (line 135) and `_find_intersect_fracs` (line 153) turn the inputs into tuples. So after creating `t_ls`, `t_le`, etc., these would be the only functions I pass the tuples into. Does that seem right?